packages/p2p/src/components/orders/chat/chat-header.jsx

```diff
@@ -31,17 +33,15 @@ const ChatHeaderBody = observer(() => {
                 >
                     {other_user_details.name}
                 </Text>
-                {sendbird_store.last_other_user_activity && (
-                    <Text
-                        as='p'
-                        className='order-chat__header-user-timestamp'
-                        color='less-prominent'
-                        line_height='m'
-                        size={isMobile() ? 'xxs' : 'xs'}
-                    >
-                        {sendbird_store.last_other_user_activity}
```

Comment:
please remove `last_other_user_activity` from `sendbird_store` also if we are not using it anywhere